3.5/runtime/windowsservercore-2004/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# escape=`
+
+FROM mcr.microsoft.com/windows/servercore:2004
+
+# Enable detection of running in a container
+ENV DOTNET_RUNNING_IN_CONTAINER=true `
+    # Ngen workaround: https://github.com/microsoft/dotnet-framework-docker/issues/231
+    COMPLUS_NGenProtectedProcess_FeatureEnabled=0
+
+COPY . .
+
+# Install .NET Fx 3.5
+RUN DISM /Online /Quiet /Add-Package /PackagePath:.\microsoft-windows-netfx3-ondemand-package~31bf3856ad364e35~amd64~~.cab `
+    && del microsoft-windows-netfx3-ondemand-package~31bf3856ad364e35~amd64~~.cab `
+    && powershell Remove-Item -Force -Recurse ${Env:TEMP}\*
+
+# ngen .NET Fx 3.5
+RUN \Windows\Microsoft.NET\Framework64\v4.0.30319\ngen update `
```

Comment:
Should this be ngen'ing Microsoft.PowerShell.Utility.Activities as well?

Reply:
Yes, adding that in.